package.json

```diff
@@ -4,15 +4,18 @@
   "description": "A repository of grist custom widgets that have no back-end requirements.",
   "scripts": {
     "build": "tsc --build && node ./buildtools/publish.js manifest.json",
-    "serve": "live-server --port=8585 --no-browser -q",
+    "serve": "http-server -c-1 --port=8585 -s",
     "build:dev": "node ./buildtools/publish.js manifest.json http://localhost:8585",
     "serve:dev": "live-server --port=8585 --no-browser -q --middleware=$(pwd)/buildtools/rewriteUrl.js",
     "watch": "nodemon --ignore manifest.json -e js,json --exec 'npm run build:dev'",
-    "dev": "echo 'Starting local server and watching for changes.\nStart Grist with an environmental variable GRIST_WIDGET_LIST_URL=http://localhost:8585/manifest.json' && npm run watch 1> /dev/null & npm run serve:dev 1> /dev/null",
+    "dev": "echo 'Starting local server and watching for changes.\nStart Grist with an environmental variable GRIST_WIDGET_LIST_URL=http://localhost:8585/manifest.json' && npm run watch 1> /dev/null & npm run serve:dev 1> /dev/null && fg",
     "test": "docker image inspect gristlabs/grist --format 'gristlabs/grist image present' && NODE_PATH=_build SELENIUM_BROWSER=chrome mocha -g \"${GREP_TESTS}\" _build/test/*.js",
     "test:ci": "MOCHA_WEBDRIVER_HEADLESS=1 npm run test",
     "pretest": "docker pull gristlabs/grist && tsc --build && node ./buildtools/publish.js manifest.json http://localhost:9998",
-    "prepack": "node ./buildtools/bundle.js --name grist-bundled --unlisted"
+    "prepack": "node ./buildtools/bundle.js --name grist-bundled --unlisted",
+    "grist": "docker run --add-host host.docker.internal:host-gateway --rm --env PORT=8484 --env GRIST_WIDGET_LIST_URL=http://host.docker.internal:8585/manifest.json -p 8484:8484 -v $PWD/_build/persist:/persist -it gristlabs/grist",
```

Comment:
I remember host.docker.internal being a problem for mac where docker is in a vm and networking is a bit inconsistent with linux but maybe that has been ironed out since? Well, we can wait for a mac user to have a concrete problem and be motivated to figure out a solution.